addons/project/models/project_task.py

```diff
@@ -1385,6 +1383,33 @@ def _task_message_auto_subscribe_notify(self, users_per_task):
                     model_description=task_model_description,
                     mail_auto_delete=False,
                 )
+            if unassigned_users:
+                task._task_message_unsubscribe_notify(unassigned_users)
```

Comment:
I think it would be better to call both methods sequentially from the `write` rather than chain them like this,

Reply:
But doing that means we will have two `for` loops for tasks that user has been change
So here we'll be doing double for loops for doing same kind of thing which can be achieved via one

May be we can change name of method? :thinking: 
Or Do you want to compromise little bit perf to gain more code readability?

Reply:
I think the overhead of a second `for` would be negligible, especially considering that you're unlikely to change followers on large recordsets

Reply:
should be fine now! :100: